tests/compile_ouroboros.py

```diff
@@ -0,0 +1,670 @@
+# -*- coding: utf-8 -*-
```

Comment:
For consistency, having this as "compile_stdlib.py" for consistency with batavia would be helpful.